specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1862,6 +1862,28 @@
         ]
       }
     },
+    "DiskEncryptionSetEncryptionType": {
+      "type": "string",
+      "description": "The type of key used to encrypt the data of resources associated with a disk encryption set.",
+      "enum": [
+        "EncryptionAtRestWithCustomerKey",
+        "EncryptionAtRestWithPlatformAndCustomerKeys"
+      ],
+      "x-ms-enum": {
+        "name": "EncryptionType",
```

Comment:
There is already EncryptionType enum in the spec.

Reply:
both are different. One has 2 values whereas the other takes 3

Reply:
there are 2 breaking changes. Adding new property and removing one enum field

Reply:
fixed the enum name